Source/Core/Core/PowerPC/SignatureDB/MEGASignatureDB.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Core/PowerPC/PPCSymbolDB.h"
+
+struct MEGASignatureReference
+{
+  u32 offset;
+  std::string name;
+
+  MEGASignatureReference(u32 _offset, std::string _name) : offset(_offset), name(_name) {}
```

Comment:
Functions (this includes constructors and destructors) should come before data members. Please done use prefixed underscores either, identifiers beginning with an underscore is reserved for use by implementations in the global namespace (even if it only has one underscore).

You can also std::move `_name` in the initializer list to prevent an additional copy.